test/simple/test-tls-dhe.js

```diff
@@ -47,6 +47,7 @@ function test(keylen, expectedCipher, cb) {
   var options = {
     key: key,
     cert: cert,
+    ciphers: ciphers,
```

Comment:
Is it actually needed? I thought that `'DHE-RSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256'` were in the default ciphers list and that this test tested that the server should work with these ciphers out of the box, without using the `ciphers` option.